unlock-app/package.json

```diff
@@ -38,31 +46,33 @@
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^4.0.0",
     "expect-enzyme": "^1.3.0",
+    "jest": "^23.6.0",
     "jest-enzyme": "^6.0.3",
     "nock": "^9.3.3",
+    "react-addons-test-utils": "15.6.2",
     "react-router-test-context": "^0.1.0",
+    "react-test-renderer": "16.4.2",
     "redux-mock-store": "^1.5.1",
-    "storybook-react-router": "^1.0.1"
+    "storybook-react-router": "^1.0.1",
+    "webpack": "^4.20.2"
   },
   "engines": {
     "node": "=8.11.4"
   },
   "engineStrict": true,
   "scripts": {
-    "start-app": "react-scripts start",
+    "dev": "npm run prestart && next src",
+    "build": "next build src",
+    "start": "next start src",
     "start-ganache": "ganache-cli -b 3 -i 1984 --deterministic &",
     "deploy-lock": "cd ../smart-contracts && truffle deploy --reset",
-    "prestart": "npm run start-ganache && npm run deploy-lock && npm run serve-unlockjs",
-    "start": "react-scripts start",
+    "prestart": "npm run start-ganache && npm run deploy-lock",
     "deploy-staging": "npm run build && TARGET=staging npm run build-unlockjs && aws s3 sync build/ s3://staging.unlock-protocol.com --profile unlock --no-progress | awk '{print $2;}' | cut -d'/' -f2- | awk '{print \"/\"$0}' | xargs aws cloudfront create-invalidation --distribution-id E1ZWU0E90B9EBN --paths --profile unlock",
-    "build": "react-scripts build",
     "build-unlockjs": "./node_modules/webpack/bin/webpack.js --config webpack-unlockjs.config.js",
-    "serve-unlockjs": "TARGET=dev node_modules/.bin/webpack-dev-server --config webpack-unlockjs.config.js &",
```

Comment:
No longer needed since it's managed by `npm run dev` aka `next`.

Reply:
ok this too must be possible to put in another PR?